docs/development.rst

```diff
@@ -22,7 +22,7 @@ Setting Up local Environement
 ============================
 Forking the Repo
 ============================
```

Comment:
because you are in the file, please replace those chapter headings by a normal heading, so we remove the exposure as main level in the index.

Reply:
I also took the liberty of rearranging them.